components/monitoring/grafana/README.md

```diff
@@ -26,7 +16,7 @@ Users must belong to the [Red Hat Appstudio SRE organization](https://github.com
 Create the secret with the following commands:
 
 ```
-$ ./hack/setup-monitoring.sh oauth2-secret grafana-oauth2-proxy $GRAFANA_GITHUB_CLIENT_ID $GRAFANA_GITHUB_CLIENT_SECRET $GRAFANA_GITHUB_COOKIE_SECRET
+$ ./hack/setup-monitoring.sh grafana-oauth2-secret $GRAFANA_GITHUB_CLIENT_ID $GRAFANA_GITHUB_CLIENT_SECRET $GRAFANA_GITHUB_COOKIE_SECRET    
 ```
 
 The `GRAFANA_GITHUB_CLIENT_ID`/`GRAFANA_GITHUB_CLIENT_SECRET` value pair must match an existing "OAuth Application" on GitHub - see [OAuth apps](https://github.com/organizations/redhat-appstudio-sre/settings/applications) in the [Red Hat Appstudio SRE organization](https://github.com/organizations/redhat-appstudio-sre). 
```

Comment:
I just checked and I can confirm that https://github.com/organizations/redhat-appstudio-sre/settings/applications is a valid URL, so I assume you're getting a 404 response because you don't have access to the settings of the https://github.com/redhat-appstudio-sre org

Reply:
makes sense. is there any note about how or who can get access it?

Reply:
You can request access to the org in Slack, in #wg-developer-stonesoup. Include the name of the team you're on if it's not obvious.